samples/common/mcumgr_bt_ota_dfu/Kconfig

```diff
@@ -138,6 +138,7 @@ config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_VALIDATION
 
 config NCS_SAMPLE_MCUMGR_BT_OTA_DFU_MCUBOOT_DIRECT_XIP_SUPPORT
 	bool
+	depends on NCS_SAMPLE_MCUMGR_BT_OTA_DFU
```

Comment:
Can you simply move this configuration option to the beginning of the `if NCS_SAMPLE_MCUMGR_BT_OTA_DFU` scope?